### ISSUE TITLE
OpenCV version

### ISSUE THREAD

**Issue by LLB8** [yorsh87/nicp#16]
Hi, 

I would like to use the lib with ROS. However ROS use the version 3.2.0 of OpenCV and I can't compile the library because it needs the version 2.4.8. 
I wonder why this version is necessary. Is there a way for me to work with it anyway ? 

LLB

**Comment by yorsh87** [comment]
I do not think it is necessary, unfortunately I do not have much time for constant updates. I will try to add this in my bucket of things to do.
